harness/determined/common/api/authentication.py

```diff
@@ -188,6 +188,7 @@ def logout(
     master_address: Optional[str],
     requested_user: Optional[str],
     cert: Optional[certs.Cert],
+    all_users: bool = False,
```

Comment:
Why change a function called `logout` which already takes a (single) `requested_user` in order to be able to magically iterate through all users?  That sounds like a really different functionality,  let's make a separate function called `logout_all` that uses `logout`.  The result is that you have two functions which do two separate things, and whose names closely reflect what they do.  And your higher-level functionality (`logout_all`) is a composition of functionality.  Composing functions to achieve DRY is often cleaner than jamming extra functionality into a single function to achieve DRY.

```python
# logout is unchanged

def logout_all(master_address, cert):
   master_address = master_address or util.get_default_master_address()
   token_store = TokenStore(master_address)

   for user in token_store.get_all_users():
       logout(master_address, user, cert)
```

Reply:
Good point! I haven't thought of before. Will do.